samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/ManageController.cs

```diff
@@ -93,6 +95,121 @@ public ActionResult VerifyAuthenticator()
             return View();
         }
 
+        public async Task<ActionResult> VerifyWebAuthnAuthenticator()
```

Comment:
I agree we can do better but didn't want to make so many changes now to avoid complex conflicts. We can create a JIRA for tech debt (there's actually a JIRA already for that) and work on this once all our work has been integrated.